samples/browser/src/main/java/org/mozilla/samples/browser/ExternalAppBrowserActivity.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.browser
+
+import androidx.core.net.toUri
+import androidx.fragment.app.Fragment
+import mozilla.components.concept.engine.manifest.WebAppManifest
+import mozilla.components.feature.pwa.ext.getWebAppManifest
+
+class ExternalAppBrowserActivity : BrowserActivity() {
```

Comment:
nit: This class deserves a comment explaining a bit what an "external app browser" is :)